Sprint-1/errors/2.js

```diff
@@ -1,5 +1,5 @@
 // Currently trying to print the string "I was born in Bolton" but it isn't working...
 // what's the error ?
```

Comment:
Try writing in your explanation as well as fixing the code - we are communicating about code here.